src/agent.rs

```diff
@@ -481,16 +480,20 @@ impl AgentInner {
     /// Send a request with data appended
     async fn send_request_append_data(
         &self,
-        request: Request,
+        mut request: Request,
         lm: &[&LocalMemoryRegion],
     ) -> io::Result<ResponseKind> {
-        let (send, recv) = oneshot::channel();
+        let (tx, mut rx) = channel(2);
```

Comment:
As there're too many kinds of channels, specify that it's oneshot is more clear.

Reply:
Thanks for the advice, but for now it will all be MPSC, because oneshot does not meet the needs of LockFreeHashMap.

Reply:
So put the comment for the reason of the channel choice. So the code reader can understand it.